_tour/pattern-matching.md

```diff
@@ -140,6 +246,20 @@ def findPlaceToSit(piece: Furniture): String = piece match {
   case b: Chair => "Sit on the chair"
 }
 ```
+{% endtab %}
+{% tab 'Scala 3' for=pattern-matching-7 %}
+```scala
+sealed abstract class Furniture
+case class Couch() extends Furniture
+case class Chair() extends Furniture
```

Comment:
I suggest taking the opportunity to make this example more idiomatic by using `case objects` instead of `case classes`.

Reply:
shouldn't this also be an enum?

Reply:
The goal is to explain `sealed` here, so we can’t use an enum. Maybe we want to use an example where the leaves are parameterized classes?